analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -322,7 +320,7 @@ def get_argparser_ctor_args():
 
     return {
         'prog': 'CodeChecker parse',
-        'formatter_class': argparse.ArgumentDefaultsHelpFormatter,
+        'formatter_class': arg.RawDescriptionDefaultHelpFormatter,
```

Comment:
Wouldn't this change modify the help output format for the parse command for the other arguments?

Reply:
By default it will use the same argument formatter: RawDescriptionHelpFormatter. If the text parameter starts with 'R|' it will keep whitespaces and it will wrap the content:
https://github.com/Ericsson/codechecker/blob/4fc14e77eb2ab00f414401c5a6af12d699d9e21c/analyzer/codechecker_analyzer/arg.py#L13-L34